<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Producto</class>
 <widget class="QDialog" name="Producto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Producto</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>E:/Descargas/producto.png</normaloff>E:/Descargas/producto.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>Producto</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>361</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Proovedor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Proovedor"/>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Precio Oferta</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="PrecOferta">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Precio C/Iva  </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="PrecIva"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Saldo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Saldo">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Precio Neto</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Precneto"/>
      </item>
     </layout>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Precio Ult. Compra</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="PrecCompra">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Descripción</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Descripcion">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>102</y>
     <width>290</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QCheckBox" name="Ofertacheck">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>310</y>
     <width>72</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Oferta</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Eliminar">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>E:/Descargas/Borrar.png</normaloff>E:/Descargas/Borrar.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Guardar">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>E:/Descargas/Guardar.png</normaloff>E:/Descargas/Guardar.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Código</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Codigo">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>E:/Descargas/producto.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
